doc/Makefile

```diff
@@ -42,18 +46,27 @@ info: $(addprefix ../info/,$(SGMLS:.sgml=.info))
 ../info/%.info: %.sgml | ../info
 	@cd ../info && linuxdoc -B info ../doc/$<
 
+../tex/%.tex: %.sgml | ../tex
+	@cd ../tex && linuxdoc -B latex ../doc/$<
+
+../tex/%.dvi: ../tex/%.tex | ../tex
+	@cd ../tex && (latex $< && latex $< && latex $< && latex $<)
+
+../pdf/%.pdf: ../tex/%.dvi | ../pdf
+	@cd ../pdf && dvipdf $<
+
 clean:
-	$(RM) -r ../html ../info
+	$(RM) -r ../html ../info ../tex ../pdf
 
 install:
 	$(if $(PREFIX),,$(error variable `PREFIX' must be set))
 ifeq ($(wildcard ../html),../html)
 	$(INSTALL) -d $(DESTDIR)$(htmldir)
 	$(INSTALL) -m0644 ../html/*.* $(DESTDIR)$(htmldir)
 endif
-ifeq ($(wildcard ../info),../info)
-	$(INSTALL) -d $(DESTDIR)$(infodir)
-	$(INSTALL) -m0644 ../info/*.* $(DESTDIR)$(infodir)
+ifeq ($(wildcard ../tex),../tex)
+	$(INSTALL) -d $(DESTDIR)$(texdir)
+	$(INSTALL) -m0644 ../tex/*.* $(DESTDIR)$(texdir)
```

Comment:
Are you positively sure that this section should *REPLACE* the "info section" (and not rather be an additional section)?

Reply:
No, definitely not. It's a search&replace artifact. Sorry, I noticed it myself only after submitting the PR. I rarely run "install".